site/frontend/src/pages/status_new/collector.vue

```diff
@@ -149,7 +149,7 @@ function ActiveStatus({collector}: {collector: CollectorConfig}) {
           <template v-for="job in collector.jobs">
             <tr v-if="ACTIVE_FILTERS[job.status]">
               <td class="table-cell-padding">
-                {{ job.requestTag }}
+                {{ job.requestTag.slice(0, 12) }}
```

Comment:
This won't really work for beta releases, which look something like `1.91.0-beta.3` and just barely won't make 12 chars. Let's create a shared function for rendering the tag that will only shorten it if it's a 40-length string (a commit SHA).